packages/client/src/interfaces/eip1193/EIP1193Provider.test.ts

```diff
@@ -30,7 +30,7 @@ describe('EIP1193Provider', () => {
     (MWPClient.createInstance as jest.Mock).mockResolvedValue(mockClient);
 
     provider = new EIP1193Provider({
-      metadata: { appName: 'Test App', appDeeplinkUrl: 'test://deeplink' },
+      metadata: { appName: 'Test App', appCustomScheme: 'test://deeplink' },
```

Comment:
If we're publishing a new version do we really need the `app` prefix on these vars?

can we just same `name`, `scheme`, etc...? feels a tad verbose